networkx/algorithms/tree/junction_tree_algorithm.py

```diff
@@ -0,0 +1,81 @@
+r"""Function for computing a junction tree of a graph."""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+from networkx.algorithms import moral, complete_to_chordal_graph, chordal_graph_cliques
+from itertools import combinations
+
+__all__ = ["junction_tree"]
+
+
+@not_implemented_for("multigraph", "MultiDiGraph")
+def junction_tree(G):
+    r"""Returns a junction tree of a given graph.
+
+    Parameters
+    ----------
+    G : networkx.Graph
+        Directed graph
```

Comment:
The input can be an undirected graph as well, correct? The description here isn't consistent with the implementation.

Reply:
You're correct, I updated the docstring accordingly.